app/views/support_tickets/start.html.erb

```diff
@@ -13,6 +13,10 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      Ordering is currently closed for devices and internet access.
```

Comment:
Please remove 'currently'

Reply:
Thanks Kate – will make this change in separate PR.